package.json

```diff
@@ -14,7 +14,7 @@
     "jquery": "2.1.4",
     "jquery-flot": "0.8.3",
     "kubernetes-container-terminal": "1.0.3",
-    "org-kubernetes-object-describer": "https://github.com/kubernetes-ui/object-describer#v1.0.4",
+    "object-describer": "git://github.com/kubernetes-ui/object-describer.git#v1.0.4",
```

Comment:
Usually I try to leave this at https instead of git, as usually that is more widely available. Does that really make a difference?